client/js/components/month/month-eventcontainer.js

```diff
@@ -1,18 +1,18 @@
 /** @jsx React.DOM */
 var React = require('react');
-var Event = require('./month-event.js');
+var MonthWeek = require('./month-week-board.js');
```

Comment:
This line seems to deviate from your previous naming convention. I would expect:

```var MonthWeekBoard = require('./month-week-board.js');```